release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m

```diff
@@ -392,7 +392,7 @@ -(void) create:(JsonObject*)json
         uiFont = [UIFont systemFontOfSize:fontSize];
     }
 
-    NSString* contentType;
+    UITextContentType* contentType = nil;
```

Comment:
I think this would now have one asterisk too many. `UITextContentType` already is a pointer type.

Reply:
Fixed.